packages/taquito/src/operations/operations.ts

```diff
@@ -113,11 +76,14 @@ export class Operation {
       throw new InvalidOperationHashError(`Invalid Operation Hash: ${this.hash}`);
     }
 
-    this.confirmed$.pipe(first(),
-      catchError(() => {
-        return of(EMPTY)
-      })
-    ).subscribe();
+    this.confirmed$
+      .pipe(
+        first(),
+        catchError(() => {
+          return of(EMPTY);
```

Comment:
EMPTY emits a complete notification. It was a fix to avoid unhandled errors propagating outside the class (related to this issue: [ropagated outside the class](https://github.com/ecadlabs/taquito/issues/1040))